.github/scripts/generate_manifests.bash

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Script for the initial generation of manifest.yaml files in inputs directories
+
+traverse_dir() {
+  local dir="$1"
+  local contains_subdir=false
+
+  # Loop through the directory contents
+  for file in "$dir"/*; do
+    if [ -d "$file" ]; then
+      contains_subdir=true
+      traverse_dir "$file"
+    fi
+  done
+
+  # If the directory does not contain any subdirectories...
+  if [ "$contains_subdir" = false ]; then
+    echo "Working on $dir:"
```

Comment:
This whole portion I think could be simply substituted by:

```suggestion
  dir_without_subdir=$(find $1 -type d -links 2 ! -empty)
  echo "Working on $dir_without_subdir:"
  ...
```

Reference:   https://unix.stackexchange.com/questions/68577/find-directories-that-do-not-contain-subdirectories/203991#203991